src/json_py.py

```diff
@@ -0,0 +1,62 @@
+import pyodbc
+import os
+from dotenv import load_dotenv
+
+class DatabaseConnector:
+    def __init__(self, env_file=".env"):
+        """Initialize connection using environment variables"""
+        # Load environment variables from .env file
+        load_dotenv(env_file)
+        
+        # Get connection details from environment
+        self.server = os.getenv("SERVER")
+        self.database = os.getenv("DATABASE")
+        self.username = os.getenv("UID")
+        self.password = os.getenv("password_value")
+        
+        # Validate all required environment variables are set
+        if not all([self.server, self.database, self.username, self.password]):
+            raise ValueError("Missing required database connection parameters in environment")
+            
+        # Establish connection
+        self.conn = pyodbc.connect(
+            f'DRIVER={{ODBC Driver 17 for SQL Server}};'
+            f'SERVER={self.server};'
+            f'DATABASE={self.database};'
+            f'UID={self.username};'
+            f'PWD={self.password}'
+        )
+        self.cursor = self.conn.cursor()
+    
+    def execute_query(self, query, params=None):
```

Comment:
No comments above methods. need to state what this method does. please explain why execute_query method and fetch_data method are separate when most of the code is similar